client/my-sites/site-settings/protect.jsx

```diff
@@ -0,0 +1,176 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { includes, trimEnd } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+import JetpackModuleToggle from 'my-sites/site-settings/jetpack-module-toggle';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import FormTextarea from 'components/forms/form-textarea';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import {
+	isJetpackModuleActive,
+	isJetpackModuleUnavailableInDevelopmentMode,
+	isJetpackSiteInDevelopmentMode
+} from 'state/selectors';
+import InfoPopover from 'components/info-popover';
+import ExternalLink from 'components/external-link';
+import QueryJetpackConnection from 'components/data/query-jetpack-connection';
+
+class Protect extends Component {
+	handleAddToWhitelist = () => {
+		const { setFieldValue } = this.props;
+		let whitelist = trimEnd( this.getProtectWhitelist() );
+
+		if ( whitelist.length ) {
+			whitelist += '\n';
+		}
+
+		setFieldValue( 'jetpack_protect_global_whitelist', whitelist + this.getIpAddress() );
+	}
+
+	getIpAddress() {
+		if ( window.app && window.app.clientIp ) {
+			return window.app.clientIp;
+		}
+
+		return null;
+	}
+
+	getProtectWhitelist() {
+		const { jetpack_protect_global_whitelist } = this.props.fields;
+		return jetpack_protect_global_whitelist || '';
+	}
+
+	isIpAddressWhitelisted() {
+		const ipAddress = this.getIpAddress();
+		const whitelist = this.getProtectWhitelist().split( '\n' );
+		return ipAddress && includes( whitelist, ipAddress );
+	}
+
+	render() {
+		const {
+			isRequestingSettings,
+			isSavingSettings,
+			onChangeField,
+			protectModuleActive,
+			protectModuleUnavailable,
+			selectedSiteId,
+			translate
+		} = this.props;
+
+		const ipAddress = this.getIpAddress();
+		const isIpWhitelisted = this.isIpAddressWhitelisted();
+		const disabled = isRequestingSettings || isSavingSettings || protectModuleUnavailable || ! protectModuleActive;
+
+		return (
+			<div>
+				<QueryJetpackConnection siteId={ selectedSiteId } />
+
+				<Card className="protect__card site-settings__security-settings">
+					<FormFieldset>
+						<div className="protect__info-link-container site-settings__info-link-container">
+							<InfoPopover position={ 'left' }>
+								<ExternalLink href={ 'https://jetpack.com/support/protect' } target="_blank">
+									{ translate( 'Learn more about Jetpack protect' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'Learn more about Jetpack. ' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).